features/filesystem/sd/SDBlockDevice.h

```diff
@@ -127,7 +127,7 @@ class SDBlockDevice : public BlockDevice {
      */
     virtual void debug(bool dbg);
 
-private:
+protected:
```

Comment:
I dont think that all members here should be protected ? 

There's a comment by @geky in the issue referenced, what is missing in the public interface that a child would like to overwrite?